scripts/ci/lib.sh

```diff
@@ -1397,8 +1404,22 @@ send_slack_failure_summary() {
     fi
 }
 
+_do_we_at_the_author() {
+    mention_author="false"
```

Comment:
How does this work? Are you saying it's modifying a `local` variable in the caller function?! 😱

Reply:
Yeah, that is how most of the slack stuff 'works'. I'll rename to your suggestion.